avocado_vt/plugins/vt_joblock.py

```diff
@@ -105,6 +105,11 @@ def pre(self, job):
                 self._lock(job)
         except Exception as detail:
             msg = "Failure trying to set Avocado-VT job lock: %s" % detail
+            files = list(self._get_lock_files())
```

Comment:
@ldoktor I agree with the overall changes, but this brings a problem.

I tested setting a lockdir to an unwritable location, and since `_get_lock_files()` is being used outside the `try/except` block, it fails to lock and we get two concurrent VT jobs running.

I would either try to simplify the exception handling by not calling anything mildly complex such as `_get_lock_files()` or just leave the UI as is.  The error condition is supposed to not be frequent, and IMHO, an Avocado-VT user usually has the ability to figure out the lock files or dir based on the current message and configuration.